test-framework/src/main/kotlin/com/epam/drill/e2e/plugin/pluginRun.kt

```diff
@@ -115,7 +115,12 @@ fun <PS : PluginStreams> E2EPluginTest.pluginRun(
     }
 }
 
-internal suspend fun waitForBuildOnline(ui: AdminUiChannels, buildVersion: String) = repeat(2) {
+internal suspend fun waitForBuildOnline(ui: AdminUiChannels, buildVersion: String) {
     while (ui.getBuild()?.let { it.buildVersion to it.buildStatus } != buildVersion to BuildStatus.ONLINE)
```

Comment:
1.Why not `while (ui.getBuild()?.buildStatus != BuildStatus.ONLINE)`
2.Function can be replaced with `waitUntil { ui.getBuild()?.buildStastus == BuildStatus.ONLINE }`

Reply:
Done